package.json

```diff
@@ -60,5 +61,8 @@
     }
   ],
   "repository": "zendesk/zcli",
-  "types": "lib/index.d.ts"
+  "types": "lib/index.d.ts",
+  "publishConfig": {
+    "registry": "https://zdrepo.jfrog.io/zdrepo/api/npm/npm"
```

Comment:
i don't think we should hardcode this here, we publish for both public and internal, which means we should dynamically pass this value during the release https://github.com/lerna/lerna/tree/main/commands/publish#--registry-url